internal/schema2/hv_socket_service_config.go

```diff
@@ -19,4 +19,6 @@ type HvSocketServiceConfig struct {
 
 	//  If true, HvSocket will process wildcard binds for this service/system combination.  Wildcard binds are secured in the registry at  SOFTWARE/Microsoft/Windows NT/CurrentVersion/Virtualization/HvSocket/WildcardDescriptors
 	AllowWildcardBinds bool `json:"AllowWildcardBinds,omitempty"`
+
+	Disabled bool `json:"Disabled,omitempty"`
```

Comment:
Could you add a comment for what this is disabling?

Reply:
Sure thing